README.md

```diff
@@ -1,6 +1,6 @@
 # Reaction Component Library & Design System
 
-[![npm (scoped)](https://img.shields.io/npm/v/@reactioncommerce/components.svg)](https://www.npmjs.com/package/@reactioncommerce/catalyst)
+[![npm (scoped)](https://img.shields.io/npm/v/@reactioncommerce/catalyst.svg)](https://www.npmjs.com/package/@reactioncommerce/catalyst)
  [![CircleCI](https://circleci.com/gh/reactioncommerce/catalyst.svg?style=svg)](https://circleci.com/gh/reactioncommerce/catalyst)
 
 ![Reaction Design System](https://blog.reactioncommerce.com/content/images/2018/09/style-guide-artwork.jpg)
```

Comment:
Add this to Line 7 for now: (we can remove it when we actually _release_ this)

> ⚠️ Warning: This repository is in active development and is not ready for testing or usage.

Line 8:

This is a single project with a package of operator-focused React UI components and code for the Catalyst Design System website: